### DISCUSSION TITLE
Future of the package

### DISCUSSION THREAD

**Discussion by elken** [radian-software/apheleia#93] [General]
Per a discussion linked in another PR, this thread was brought up:

https://github.com/lassik/emacs-format-all-the-code/issues/170

I've gone through and it seems like there's universal agreement to merge the packages under one umbrella.

[Doom](https://github.com/doomemacs/doomemacs) currently depends on `format-all` as its formatter tool, I've recently started a refactor to use `apheleia` instead; but now I'm unsure if it would be worth committing all-in now if there will be a future plan to create a "super-formatter" package.

Keen to get your thoughts @raxod502.

**Comment by raxod502** [comment]
There will certainly be a future plan to merge these packages, but I do not think it would necessarily be worth waiting for, as I don't think we have a firm timeline on that.

**Reply by elken** [reply]
Thanks, I'll carry on as I am then :) If/when that does progress, it would be useful for the discussion to be public but I consider this resolved now.

**Reply by raxod502** [reply]
Yep, I certainly plan for that discussion to be public. Feel free to follow the linked issue and I should be posting any updates there.
